services/simplified_crontab

```diff
@@ -45,6 +45,9 @@ HOME=/var/www/circulation
 # If any works have out-of-date OPDS entries, rebuild them,
 40 22 * * * root core/bin/run opds_entry_coverage >> /var/log/cron.log 2>&1
 
+# Sync a library's collection with NoveList
+0 0 * * 0 root core/bin/run novelist_update >> /var/log/cron.log 2>&1
+
```

Comment:
I have to look up the cron format every time. I'm assuming that last 0 is for day-of-the-week, but http://www.nncron.ru/help/EN/working/cron-format.htm says that day-of-the-week goes from 1 to 7, so I'd change this to 1.

Reply:
I was following this https://www.pantz.org/software/cron/croninfo.html which says it's 0-6. Wikipedia says the same and this is pretty cool (which also says 0-6): https://crontab.guru/#0_0_*_*_0

Reply:
OK, we're cool then.